<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creditCalc</class>
 <widget class="QDialog" name="creditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
creditCalc {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0  #E6E6FA, stop:1  #FFB6C1); /* Устанавливаем цвет фона для главного окна */
}

creditCalc * {
    background: transparent; /* Сбрасываем цвет фона для всех дочерних элементов главного окна */
}</string>
  </property>
  <widget class="QPushButton" name="PopButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
 background: #E6E6FA; border-radius: 12px; border: 2px solid white; 
}
QPushButton:pressed {
   background: #cfd2fa; border-radius: 12px; border: 2px solid white; 
}</string>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>581</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string>QDoubleSpinBox {
    border: 2px solid pink; /* цвет и толщина границы */
    border-radius: 5px; /* радиус скругления границы */
    padding: 2px 5px; /* отступ вокруг контейнера */
   background-color: #f0f0f0; /* цвет фона */
}
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_totalLoanAmount">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox {
background-color: white;
}</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_totalLoanAmount">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #E6E6FA; border-radius: 12px; border: 2px solid white;</string>
      </property>
      <property name="text">
       <string>Total loan amount</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_term">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #E6E6FA; border-radius: 12px; border: 2px solid white;</string>
      </property>
      <property name="text">
       <string>Term (in months)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_interestRate">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #E6E6FA; border-radius: 12px; border: 2px solid white;</string>
      </property>
      <property name="text">
       <string>Interest rate (%)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_interestRate">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox {
background-color: white;
}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_term">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox {
background-color: white;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
 background: #E6E6FA; border-radius: 12px; border: 2px solid blue; 
}
QPushButton:pressed {
   background: #cfd2fa; border-radius: 12px; border: 2px solid white; 
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_monthlyPayment">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #E6E6FA; border-radius: 12px; border: 2px solid white;</string>
      </property>
      <property name="text">
       <string>Monthly payment</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_totalPayment">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #E6E6FA; border-radius: 12px; border: 2px solid white;</string>
      </property>
      <property name="text">
       <string>Total payment</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_loanOverpayment">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #E6E6FA; border-radius: 12px; border: 2px solid white;</string>
      </property>
      <property name="text">
       <string>Loan overpayment</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>315</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_typeCreadit">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_annuity">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
    spacing: 5px; /* отступ между кружочком и текстом */
}

QRadioButton::indicator {
    width: 15px; /* ширина радиокнопки */
    height: 15px; /* высота радиокнопки */
    border: 2px solid #8079c6;; /* цвет и толщина границы */
    border-radius: 7.5px; /* радиус скругления кружочка */
}

QRadioButton::indicator:checked {
    background-color: #8079c6;; /* цвет заливки для выбранной радиокнопки */
    border: 2px solid #8079c6;; /* цвет и толщина границы для выбранной радиокнопки */
}
</string>
      </property>
      <property name="text">
       <string>annuity</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_differentiated">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton {
    spacing: 5px; /* отступ между кружочком и текстом */
}

QRadioButton::indicator {
    width: 15px; /* ширина радиокнопки */
    height: 15px; /* высота радиокнопки */
    border: 2px solid #8079c6;; /* цвет и толщина границы */
    border-radius: 7.5px; /* радиус скругления кружочка */
}

QRadioButton::indicator:checked {
    background-color: #8079c6;; /* цвет заливки для выбранной радиокнопки */
    border: 2px solid #8079c6;; /* цвет и толщина границы для выбранной радиокнопки */
}
</string>
      </property>
      <property name="text">
       <string>differentiated</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_monthlyPayment_res">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>381</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_totalPayment_res">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>405</y>
     <width>381</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_loanOverpayment_res">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>454</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
